composer.json

```diff
@@ -11,10 +16,10 @@
         }
     ],
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.1",
         "php-http/httplug": "^2.0",
         "php-http/discovery": "^1.0",
-        "cakephp/cakephp": "^4.2"
+        "cakephp/cakephp": "5.x-dev"
```

Comment:
will we revert this before merging?

Reply:
Of course, this is only until cake5 stable is released.